builtin/providers/clc/resource_clc_server.go

```diff
@@ -44,15 +44,14 @@ func resourceCLCServer() *schema.Resource {
 				Type:     schema.TypeInt,
 				Required: true,
 			},
+			// optional
 			"password": &schema.Schema{
 				Type:     schema.TypeString,
-				Required: true,
+				Optional: true,
```

Comment:
To add, the clc_sdk will omit [1] an empty password. 

[1] https://github.com/CenturyLinkCloud/clc-sdk/blob/master/server/server.go#L327